app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UserSerializer
+  include JSONAPI::Serializer
+
+  attributes :email, :full_name, :avatar_url, :provider, :provider_uid
```

Comment:
For users, `provider` and `provider_uid` seems redundant IMO.

Reply:
Hmmmm, are you suggesting we should remove it from this serializer class or from the table? 💭

Reply:
At this stage, I guess we don't need `provider` at all

However, if we think ahead and start adding `twitter` for example then we should keep it 👍

Reply:
@Tuubz I meant for the response we don't need those fields 😅 For now, I think we could leave it as it is 👌